collect_app/src/main/java/org/odk/collect/android/utilities/FormEntryPromptUtils.java

```diff
@@ -124,4 +126,13 @@ public static String markQuestionIfIsRequired(String questionText, boolean isReq
 
         return questionText;
     }
+
+    public static Optional<String> getAttributeValue(FormEntryPrompt prompt, String attributeName) {
```

Comment:
We talked briefly about using optionals vs sticking to nulls. Are we diving into optionals?

Reply:
Ah yeah good point. I'm in two minds. I think your point below is a good one. Java Optionals really don't add much as calling `get` is super easy. I think I'm happy to convert the `Optional` from the stream into a normal type and using `Nullable`.